Gemfile

```diff
@@ -26,6 +26,7 @@ group :development, :test do
   gem 'database_cleaner'
   gem 'capybara'
   gem 'launchy'
+  gem 'poltergeist'
```

Comment:
Why did you add this gem, I can't see you using the js enable driver in your tests.